src/number_format.js

```diff
@@ -536,9 +538,16 @@ class NumberFormat extends React.Component {
   }
 
   formatNumString(numStr: string = '') {
-    const { format, allowEmptyFormatting } = this.props;
+    const { format, allowEmptyFormatting, customNumerals } = this.props;
     let formattedValue = numStr;
 
+    if (customNumerals && customNumerals.length === 10) {
```

Comment:
Can we add a proptype check for length of customNumerals.?

Reply:
i wrote a custom Proptype method in 79860f4